quantum/process_keycode/process_combo.c

```diff
@@ -175,3 +193,28 @@ void matrix_scan_combo(void) {
     dump_key_buffer(true);
   }
 }
+
+void combo_enable(void) {
+    is_combo_enable = true;
+}
+
+void combo_disable(void) {
+    is_combo_enable = false;
+    is_active = false;
+    timer = 0;
+    dump_key_buffer(true);
+
+}
+
+void combo_toggle(void) {
+    is_combo_enable ^= true;
```

Comment:
I would be tempted to reduce duplication here
```suggestion
    is_combo_enable ? combo_disable() : combo_enable();
```

Reply:
Yeah, that's probably simpler.